CvGameCoreDLL_Expansion2/CvGame.cpp

```diff
@@ -4383,26 +4385,32 @@ bool CvGame::canTrainNukes() const
 
 //	--------------------------------------------------------------------------------
 EraTypes CvGame::getCurrentEra() const
+{
+	{
+		return m_eGameEra;
+	}
+}
+
+void CvGame::UpdateGameEra()
 {
 	float fEra = 0;
 	int iCount = 0;
-
-	for(int iI = 0; iI < MAX_TEAMS; iI++)
+	for (int iI = 0; iI < MAX_TEAMS; iI++)
 	{
 		if (GET_TEAM((TeamTypes)iI).isAlive() && GET_TEAM((TeamTypes)iI).isMajorCiv())
 		{
```

Comment:
@Mileslewis It looks like fEra will always be 0? How does this code work?

Reply:
Yea good spot, must've been a merge error. Fixed now.